### DISCUSSION TITLE
problem in updating bootstrap v3.3.1 to v4.5.3

### DISCUSSION THREAD

**Discussion by ghost** [twbs/bootstrap#32252] [Q&A] [answered]
in eclipse we have a jquery version 3.5.1 i'm trying to update bootstrap version to v4.5.3 but in bootstap.min.js i'm facing so many errors like invalid syntax error 'default' identifier expected can someone help me on this issue.
thanks in advance

**Comment by mdo** [comment, marked as answer]
v3 and v4 (and now v5) have entirely different codebases. They're not drop-in replacements. You'll need to read those migration guides for each major version.
